src/backends/caffe/caffelib.cc

```diff
@@ -2304,6 +2419,65 @@ namespace dd
 		  vrad.push_back(outseq);
 		}
 	    }
+           else if (typeid(inputc) == typeid(CSVTSCaffeInputFileConn)) // timeseries
+             {
+               int slot = findOutputSlotNumberByBlobName(_net,"OUTPUT");
+               //results[slot] is TxNxDataDim , N is batchsize ...
+               int nout = _ntargets;
+
+               const boost::shared_ptr<Blob<float>> contseq = _net->blob_by_name("cont_seq");
+               //cont_seq is TxN
+
+               CSVTSCaffeInputFileConn* ic =
+                 reinterpret_cast<CSVTSCaffeInputFileConn*>(&inputc);
+
+               for (int j=0; j<batch_size; ++j)
+                 {
+                   for (int t=0; t<ic->_timesteps; ++t)
+                     {
+                       std::vector<int> loc0 = {t,j};
+                       if (contseq->data_at(loc0) == 0 && series.size() != 0)
+                         // new seq -> push old one
+                         {
+                           if (series.size() > 1)
+                             {
+                               APIData out;
+                               if (!inputc._ids.empty())
+                                 out.add("uri",inputc._ids.at(serieNum++));
+                               else out.add("uri",std::to_string(serieNum++));
+                               out.add("series", series);
+                               out.add("probs",std::vector<double>(series.size(),1.0));
```

Comment:
Do we get confidences for every timestep prediction ? That'd be great! If only useful part of the time we may want to (later) reuse the `confidences` API parameter from PR #530

Reply:
no we dont, we are in regression mode, and nothing that can be used confidence comes out from the LSTM layers. Confidence can be simulated with dropouts (to see possible amount of variation, and variational version of lstm, if they exist :)

Reply:
Right, do we need to return `probs` ?

Reply:
in supervisedouputconnector::add_results (l.190) , results are iterated based on probs.size(), and stuff like (prob[i], result[i]) are added to _vvcats, so not putting probs needs a bit of refactoring